<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FollowerUiClass</class>
 <widget class="QMainWindow" name="FollowerUiClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Follower</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="BtnSeartchPorts">
          <property name="text">
           <string>Otsi seadmeid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBPort"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Andurid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_andur1">
          <property name="text">
           <string>Andur1:  0cm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_andur2">
          <property name="text">
           <string>Andur2:  0cm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_andur3">
          <property name="text">
           <string>Andur3:  0cm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_andur4">
          <property name="text">
           <string>Andur4:  0cm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_andur5">
          <property name="text">
           <string>Andur5:  0cm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_andur6">
          <property name="text">
           <string>Andur6:  0cm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_andur7">
          <property name="text">
           <string>Andur7:  0cm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_andur8">
          <property name="text">
           <string>Andur8:  0cm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_yaw">
          <property name="text">
           <string>yaw:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_pitch">
          <property name="text">
           <string>pitch:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_roll">
          <property name="text">
           <string>roll:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Kiirus (m/s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_setSpeed">
          <property name="minimum">
           <number>-65535</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_leftSpeed">
          <property name="text">
           <string>Vasak:     0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_rightSpeed">
          <property name="text">
           <string>Parem:    0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
